lib/bgsite_official_web/templates/user_registration/new.html.eex

```diff
@@ -21,6 +21,7 @@
 <% end %>
 <br>
 <p>
-  <%= link "Вход", to: Routes.user_session_path(@conn, :new) %> |
-  <%= link "Забравена парола", to: Routes.user_reset_password_path(@conn, :new) %>
+  <%= link "Вход", to: Routes.user_session_path(@conn, :new), class: "btn btn-info" %>
+  
+  <%= link "Забравена парола", to: Routes.user_reset_password_path(@conn, :new), class: "btn btn-info" %>
 </p>
```

Comment:
Maybe it would be better to have something like:
In Registration - "Регистрирай ме" за нов акаунт, "Имаш вече акаунт" за линк към sign_in.
И за Log In страницата "Регистрация" ще бъде по-добре - "Нямаш акаунт" или "Направи си профил/акаунт"
Нещо такова, според мен